src/Tests/Unit/Config/MaxLoginConfig.test.ts

```
@@ -85,23 +69,56 @@ const MOCK_LOCATOR = jest.fn().mockReturnValue({
+  MOCK_WAIT_FOR_SELECTOR,
+  MOCK_WAIT_FOR_URL,
+  MOCK_LOCATOR,
+  MOCK_GET_BY_TEXT,
+  MOCK_GET_BY_ROLE,
+];
+/**
+ * Reset all shared mocks and wire getByText/getByRole.
+ * @returns The number of mocks cleared.
+ */
+function resetMocks(): number {
+  ALL_MOCKS.forEach(m => m.mockClear());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)